repos/system_upgrade/el7toel8/actors/vimmigrate/actor.py

```diff
@@ -0,0 +1,59 @@
+from leapp.actors import Actor
+from leapp.libraries.common.reporting import report_generic
+from leapp.models import Report, VimModel
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
+
+from os import access, W_OK
+
+class VimMigrate(Actor):
+    """
+    Actor for adding two directives into vim configuration files to simulate 
+    the same behavior as vim-7.4 has.
+    """
+
+    name = 'vim_migrate'
+    consumes = (VimModel,)
+    produces = (Report,)
+    tags = (ApplicationsPhaseTag, IPUWorkflowTag)
+
+    """ Macros for adding into configuration file"""
+    _changes = [
+                'let skip_defaults_vim=1',
+                'set t_BE='
+              ]
+
+
+    def insert_string(self, string, path):
+        """
+        Insert string into a file
+
+        :param str string: string which will be inserted
+        :param str path: string representing path
+        """
+
+        if not access(path, W_OK):
+            raise OSError('File {} does not have writing permissions.'.format(path))
+
+        try:
+            with open(path, 'a') as f:
```

Comment:
You probably want to insert a new line before too just to be sure

Reply:
Yeap, we did even additional changes here. Let's wait for push force.